task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +306,15 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    let sum = Array.from(String(num)).reduce((a, b) => {
+        return a + Number(b);
+    }, 0);
+    if (sum > 9) {
+        return Array.from(String(sum)).reduce((a, b) => {
+            return a + Number(b);
+        }, 0);
+    }
```

Comment:
Here should be some kind of loop, because of test can contain very large number for those the second step will return sum with a few digits